pkg/configuration/configuration.go

```diff
@@ -151,7 +151,15 @@ func (ev *extendedViper) Clone() Configuration {
 	defer ev.mutex.Unlock()
 
 	// manually clone the Configuration instance
-	clone := NewFromFiles(ev.viper.ConfigFileUsed())
+	configFileUsed := ev.viper.ConfigFileUsed()
```

Comment:
suggestion: should we derive the type of the cloned configuration from this configuration?

Reply:
Dunno what you mean - let's chat :)
=